src/harness/parallel/worker.ts

```diff
@@ -33,19 +35,19 @@ namespace Harness.Parallel.Worker {
         }) as Mocha.ITestDefinition;
     }
 
-    function executeSuiteCallback(name: string, callback: Function) {
+    function executeSuiteCallback(name: string, callback: MochaCallback) {
         const fakeContext: Mocha.ISuiteCallbackContext = {
             retries() { return this; },
             slow() { return this; },
             timeout() { return this; },
         };
         namestack.push(name);
-        let beforeFunc: Function;
-        (before as any) = (cb: Function) => beforeFunc = cb;
-        let afterFunc: Function;
-        (after as any) = (cb: Function) => afterFunc = cb;
+        let beforeFunc: () => void;

```

Comment:
You sure you don't want to write `type Callable = () => void` somewhere and use it everywhere just to reduce the verbosity around places like here? Callback types have a lot of sigils and are visually noisy because of it.